src/main/java/org/apache/sling/feature/cpconverter/handlers/RepPrincipalPolicyEntryHandler.java

```diff
@@ -78,7 +79,7 @@ public void startElement(String uri, String localName, String qName, Attributes
                         throw new IllegalStateException("isolated principal-based access control entry. no principal found.");
                     }
                     List<String> privileges = extractValues(attributes.getValue(REP_PRIVILEGES));
-                    RepoPath effectivePath = new RepoPath(attributes.getValue(REP_EFFECTIVE_PATH));
+                    RepoPath effectivePath = new RepoPath(extractEffectivePath(attributes.getValue(REP_EFFECTIVE_PATH)));
```

Comment:
You can use https://github.com/apache/sling-org-apache-sling-feature-cpconverter/blob/e53fdfaa10cab40aea5d6d4adcb06a7ea79862cc/src/main/java/org/apache/sling/feature/cpconverter/vltpkg/ValueConverter.java#L60 instead

Reply:
Sorry, that is just for the converting the other way around, to parse enhanced docview  please use `org.apache.jackrabbit.vault.util.parse(...)`

Reply:
@kwin , fine with me.... (as long as nobody passes an empty MV property array that should work)